internal/data-ingest-cli/receiver/otlp/otlp.go

```diff
@@ -77,7 +77,7 @@ func (r *OTLPReceiver) Stop() {
 
 	select {
 	case <-r.receivedDataChan:
-	case <-time.After(10 * time.Second):
+	case <-time.After(300 * time.Second):
```

Comment:
eventually it might be good to have this timeout configurable or having an option to disable it, but I'd be fine with having this done in a separate PR

Reply:
I'll make it configurable as part of this PR

Reply:
adapted, please have a re-review